js/modules/swiperCommentsFunctionality.js

```diff
@@ -0,0 +1,127 @@
+import { setSlidesState } from "./swiperHelpers.js";
+import { switchSlidesStateHandlerOn1920px } from "./swiperHelpers.js";
+import Swiper from "./swiper-bundle.8.4.5.esm.browser.min.js";
+import { mqlArray } from "./helpers.js" ;
+
+
+const slides = document.querySelectorAll(".comments_card_container");
```

Comment:
you already have slides in swiper object, don't use querySelectorAll

Reply:
Done.